src/screens/profile/components/user-info/user-info.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { View } from 'react-native';
+import { Avatar } from 'react-native-paper';
+
+import { ShieldCheckIcon, Heading, Text } from 'src/components';
+import { AppColor, HeadingLevel } from 'src/common/enums';
+
+import styles from './styles';
+
+const UserInfo = () => {
+  return (
+    <View style={styles.container}>
+      <Avatar.Image
+        size={148}
+        source={require('src/assets/images/user_avatar.png')}
+        style={styles.avatar}
+      />
```

Comment:
We can add it using just Icon. Don't forget to remove user_avatar.png 
```suggestion
    <Avatar.Icon size={152} icon="account" style={styles.avatar} />
```